manifests/n/NathanTech/Calliope/3.6/NathanTech.Calliope.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: NathanTech.Calliope
+PackageVersion: 3.6
+Installers:
+- Architecture: neutral

```

Installs to the program files x86 directory
```suggestion
- Architecture: x86
```